src/Networks/Blockcore.Networks.x42/x42Setup.cs

```diff
@@ -50,7 +50,7 @@ internal class x42Setup
             HashGenesisBlock = "0x04ffe583707a96c1c2eb54af33a4b1dc6d9d8e09fea8c9a7b097ba88f0cb64c4",
             HashMerkleRoot = "0x6e3439a32382f83dee4f94a6f8bdd38908bcf0c82ec09aba85c5321357f01f67",
             DNS = new[] { "mainnet1.x42seed.host", "mainnetnode1.x42seed.host", "tech.x42.cloud", "x42.seed.blockcore.net" },
-            Nodes = new[] { "34.255.35.42", "52.211.235.48", "63.32.82.169", "18.179.72.204", "15.188.129.215", "18.157.117.214" },
+            Nodes = new[] { "161.97.101.147", "149.102.149.209", "121.170.70.100", "209.145.56.58", "154.26.136.64", "193.187.129.227", "51.254.85.40", "89.10.238.216", "121.170.70.22", "149.102.131.85", "152.228.148.188", "212.90.120.47" },
```

Comment:
"152.228.148.188" This is my server and it may be shut down soon. Please remove it from seeds and use servers that belong to you or you are sure that it is on.

Reply:
Done.